### PR TITLE
Update `README.md`

### PR DESCRIPTION
**Pull request by JackPGreen**

### CODE REVIEW

**Review by nishaatr — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # rel-scripts
+
+Metadata used to populate the [Hazelcast downloads page](https://hazelcast.com/get-started/download).
```
[the remark this answers is not in the document]
Comment (JackPGreen) [reply README.md]: > We also have OSS https://hazelcast.com/community-edition-projects/downloads/#hazelcast-platform

e7adf94504154bd03b5a7acb4b47774b527b7b4f

> This repo has no branch protection :(

We can address separately if required

> anyone could change this

It's internal only

> By adding this info I like we might be drawing attention

I'm just trying to meet our [repository policy](https://hazelcast.atlassian.net/wiki/spaces/DI/pages/5040668688/GitHub+repository+creation+policy#Policy).

[the remark this answers is not in the document]
Comment (nishaatr) [reply README.md]: Repo is public so prefer we set the protection

@promanenko / @yuce / @emreyigit 
I see you have made changes here. I am going to add PR requirement and only allow [devOpsHazelcast](https://github.com/hazelcast/rel-scripts/commits?author=devOpsHazelcast) to push without PR
Is there any other BOT account to include?
Let me know if its ok?

[the remark this answers is not in the document]
Comment (nishaatr) [reply README.md]: I have added branch rule for `master`

**Review by nishaatr — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JackPGreen) [reply README.md]: @nishaatr I think somethings gone wrong with branch protection:
image

[the remark this answers is not in the document]
Comment (nishaatr) [reply README.md]: odd. You are part of https://github.com/orgs/hazelcast/teams/employees?query=jack which has write perm

[the remark this answers is not in the document]
Comment (nishaatr) [reply README.md]: I removed below. Don't need it I believe 

image

[the remark this answers is not in the document]
Comment (yuce) [reply README.md]: > Repo is public so prefer we set the protection
> 
> @promanenko / @yuce / @emreyigit I see you have made changes here. I am going to add PR requirement and only allow [devOpsHazelcast](https://github.com/hazelcast/rel-scripts/commits?author=devOpsHazelcast) to push without PR Is there any other BOT account to include? Let me know if its ok?

@nishaatr I always create a PR and do not directly push to `master`. I believe other team mates would do the same.

:+1:  Protecting the `master` is good practice in any case.

[the remark this answers is not in the document]
Comment (promanenko) [reply README.md]: > @promanenko / @yuce / @emreyigit
I see you have made changes here. I am going to add PR requirement and only allow [devOpsHazelcast](https://github.com/hazelcast/rel-scripts/commits?author=devOpsHazelcast) to push without PR
Is there any other BOT account to include?
Let me know if its ok?

works for us, there is no automation for rel-scrips repo in MC

**Pull request merged**